internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```diff
@@ -0,0 +1,50 @@
+
+#### Package Parameters
+The following package parameters are available:
+
+ * `/SPLUNK_ACCESS_TOKEN` - The splunk access token (org token) used to send metric data to OTEL. If the parameter is specified, the token will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\token` file. If the parameter is not specified and `\ProgramData\Splunk\OpenTelemetry Collector\token` does not exist or is empty, the Agent service is not started after installation or upgrade. To start the service, add or update `\ProgramData\Splunk\OpenTelemetry Collector\token` with a valid token, and then either restart Windows or run the following PowerShell command: `&amp; "\ProgramData\Splunk\OpenTelemetry Collector\otelcol.exe"`
+ * `/SPLUNK_INGEST_URL:` - URL of the Splunk ingest  (e.g. `https://ingest.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_INGEST_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_INGEST_URL` (if it exists) will be used. Otherwise, defaults to `https://ingest.us0.signalfx.com`.
+ * `/PLUNK_API_URL` - URL of the API endpoint (e.g. `https://api.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` (if it exists) will be used. Otherwise, defaults to `https://api.us0.signalfx.com`.
+ * `/install_dir` - Installation directory. Defaults to `\Program Files\Splunk\OpenTelemetry Collector`.
+ * `/SPLUNK_HEC_TOKEN` - Splunk HEC is HTTP Event Collecter token which will collect the metrics and logs of host system in to splunk.. If the parameter is specified, the token will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\hec-token` file.
+ * `/SPLUNK_HEC_URL` - URL of Splunk HEC (e.g. `https://ingest.$SPLUNK_REALM.signalfx.com/v1/log`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_HEC_URL` file.
+ * `/SPLUNK_TRACE_URL` - Trace url is end point where apllication traces will be collected. URL of Splunk TRACE (e.g. `https://ingest.$SPLUNK_REALM.signalfx.com/v2/trace`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_TRACE_URL` file.
+ * `/SPLUNK_REALM` - The parameter will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_REALM` file. If not specified default is ("us0").
+ * `/SPLUNK_BUNDLE_DIR` - The path to the Agent bundle directory. The default path is provided by the collector package. If the specified path is changed from the default value, the path should be an existing directory on the node. The SPLUNK_BUNDLE_DIR environment variable will be set to this value for the collector service.
+ 

```

Comment:
The `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_*` files are not relevant for splunk-otel-collector.  The nupkg installer script should check the registry for these keys/values and create/update them appropriately based on the choco installation parameters.

Reply:
Suggested changes are made.